go.mod

```diff
@@ -24,18 +24,18 @@ require (
 	golang.org/x/net v0.0.0-20210520170846-37e1c6afe023
 	google.golang.org/grpc v1.41.0
 	google.golang.org/protobuf v1.26.0
-	k8s.io/api v0.22.3
-	k8s.io/apimachinery v0.22.3
-	k8s.io/client-go v0.22.3
-	k8s.io/cloud-provider v0.22.3
-	k8s.io/code-generator v0.22.3
-	k8s.io/component-base v0.22.3
-	k8s.io/component-helpers v0.22.3
+	k8s.io/api v0.22.4
+	k8s.io/apimachinery v0.22.4
+	k8s.io/client-go v0.22.4
+	k8s.io/cloud-provider v0.22.4
+	k8s.io/code-generator v0.22.4
+	k8s.io/component-base v0.22.4
+	k8s.io/component-helpers v0.22.4
 	k8s.io/klog v1.0.0
 	k8s.io/klog/v2 v2.10.0
-	k8s.io/kube-scheduler v0.22.3
+	k8s.io/kube-scheduler v0.22.4
 	k8s.io/kubernetes v1.22.3
```

Comment:
I wonder if we should bump `k8s.io/kubernetes` up to v1.22.4 as well.

Reply:
Done.